src/DoctrineServiceProvider.php

```diff
@@ -270,6 +275,24 @@ public function extendNotificationChannel(): void
         });
     }
 
+    /**
+     * Register the Entity factory instance in the container.
+     */
+    protected function registerEntityFactory(): void
+    {
+        $this->app->singleton(FakerGenerator::class, static function ($app) {
```

Comment:
I got the same issue and saw your PR 

This shouldn't be needed, the Generator should be solveable. 
This also prevents moving the faker dependency from the `require-dev` section. 

I got the following temporarily in my `AppServiceProvider` 

```
$this->app->singleton(Factory::class, static function ($app) {
            return Factory::construct(
                $app->make(Generator::class),
                $app->make('registry'),
                database_path('factories'),
);
});
```

This seems to do the trick for me  ;)

Reply:
I agree about the require-dev section home for faker.  Can this be solved using an instantiator?  

Gonna need to not move faker from require-dev and put some unit tests in too.